src/hooks/use-size-browser.ts

```diff
@@ -0,0 +1,27 @@
+import { useCallback, useEffect, useState } from 'react';
+
+import { useBrowserResize } from './use-browser-resize';
+
+interface SizeBrowser {
+  browserWidth: number;
+  browserHeight: number;
+}
+
+export const useSizeBrowser = (): SizeBrowser => {

```

Comment:
It seem like you can unite two your hooks. I think that it doesn`t use without useBrowserResize and it without useSizeBrowser